app/forms/declaration/DeclarationAdditionalActors.scala

```diff
@@ -72,6 +79,13 @@ object DeclarationAdditionalActors extends DeclarationPage {
 
   def form(): Form[DeclarationAdditionalActors] = Form(mapping)
 
+  def actorsValidator(actor: DeclarationAdditionalActors, actors: Seq[DeclarationAdditionalActors]): Option[Seq[(String, String)]] =
```

Comment:
Is there a specific reason/situation for this method to be here? At first glance it is more apropriate to be part of `DeclarationAdditionalActorsData` class.

Reply:
That's a good idea.